doc/cephadm/services/index.rst

```diff
@@ -86,7 +86,10 @@ Service Specification
 =====================
 
 A *Service Specification* is a data structure that is used to specify the
-deployment of services.  Here is an example of a service specification in YAML:
+deployment of services. In addition to parameters such as `placement`, etc
+the user can set initial values of service configuration parameters.
```

Comment:
I feel like this comes out a bit vague. We should probably be a bit more clear that this sets ceph config options. Basically running beneath the hood `ceph config set <service-name> <param> <value>` for each param/value pair provided and doing so declaratively (as in, we'll keep checking the setting and change it if it has the wrong value). Additionally, you may want to try and incorporate some stuff from the first attempt at documenting this https://github.com/ceph/ceph/pull/44804

Reply:
@adk3798 agree, I reworded the doc and added more information. Please, let me know if more clarification is needed.